src/Mailer/Email.php

```diff
@@ -1023,7 +1023,7 @@ protected function _constructTransport($name)
             return $config['className'];
         }
 
-        $className = App::className($config['className'], 'Network/Email', 'Transport');
+        $className = App::className($config['className'], 'Mailer/Transport', 'Transport');
```

Comment:
I think it is fine if the break as long a we have some sort of upgrade command that can help you move the classes.